app/src/main/java/com/example/recipeapp/Models/Parse/Taste.java

```diff
@@ -15,8 +21,65 @@ public class Taste extends ParseObject {
     public static final String KEY_FATTINESS = "fattiness";
     public static final String KEY_SPICINESS = "spiciness";
 
-    public void updateTaste(Recipe recipe, int numberOfVotes) {
+    private static final int NUMBER_OF_COMPONENTS = 7;
```

Comment:
Is there a way that we can make sure this number is in sync with the number of the tastes below? Otherwise, does it worth a comment to mention this that whenever we add/remove a taste, we need to increase/decrease this `NUMBER_OF_COMPONENTS`?